src/linagora.esn.resource/app/app.js

```diff
@@ -12,3 +12,38 @@
     'restangular'
   ]);
 })(angular);
+
+require('esn-frontend-common-libs/src/frontend/js/modules/attendee/attendee.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/components/dynamic-directive/src/dynamic-directive.js');
```

Comment:
Is this needed to require here? You do not require this in the calendar app.module file.